src/playback/qml/MuseScore/Playback/internal/AudioResourceControl.qml

```diff
@@ -234,6 +234,10 @@ Item {
                     width: titleLoader.width - 8 // 4px padding on each side
 
                     text: root.title
+
+                    onImplicitWidthChanged: {
+                        let i = implicitWidth // workaround for strange binding loop warning
```

Comment:
How often does this warning appear?

Because the workaround looks strange. I think either you need to deal with the warning or leave it as it is, maybe someone will find a better solution.

Reply:
It appears not crazily annoyingly often; just sometimes when a Mixer channel is loaded in the view. I've removed the workaround now in the hope that the warning inspires someone, and if it becomes too annoying we can add back the workaround later.